frontend/src/components/GrantAccess.vue

```diff
@@ -119,211 +131,298 @@ async function getForestClientNumber(forestClientNumber: string) {
+                                    name="forestClientNumber"
+                                />
+                            </div>
+                            <div class="col-md-2">
+                                <Button
+                                    class="button p-button-tertiary p-button-outlined"
+                                    @click="
+                                        getForestClientNumber(
+                                            formData.forestClientNumber
+                                        )
+                                    "
+                                    v-bind:disabled="
+                                        formData.forestClientNumber?.length <
+                                            8 || loading
+                                    "
+                                >
+                                    <div class="w-100">
+                                        <div v-if="loading">
+                                            <span> Loading... </span>
+                                        </div>
+                                        <div v-else>Verify</div>
+                                    </div>
+                                </Button>
                             </div>
-                            <div v-else>Verify ID</div>
-                        </button>
+                        </div>
+                    </div>
+                    <div class="row" v-if="forestClient">
+                        <div class="form-group col-md-5 px-0">
+                            <ForestClientCard
+                                :text="forestClient?.[0]?.client_name"
+                                :status="forestClient?.[0]?.status"
+                            ></ForestClientCard>
+                        </div>
                     </div>
-                </div>
-            </div>
-            <div class="row" v-if="forestClient">
-                <div class="form-group col-md-3">
-                    <label class="col-sm control-label"
-                        >Organization details:</label
-                    >
-                    <ForestClientCard
-                        :text="forestClient?.[0]?.client_name"
-                        :status="forestClient?.[0]?.status"
-                    ></ForestClientCard>
-                </div>
-            </div>
 
-            <div class="row gy-3">
-                <div class="col-auto">
-                    <button
-                        type="submit"
-                        id="grantAccessSubmit"
-                        class="btn btn-primary mb-3"
-                        :disabled="!meta.valid || invalidForestClient"
-                    >
-                        Grant Access
-                    </button>
-                </div>
+                    <div class="row gy-3 my-0" v-if="meta.valid">
+                        <div class="col-auto px-0">
+                            <Button
```

Comment:
It seems to me, when user enters "forestClientNumber" (it may be correct, or it may be incorrect number) and the "Verify" button help to verify the correctness. However, as soon as I enter 8 digits, this "Submit" button is active and can go to summary page (without user verifies the forestClientNumber).
Has this behavior being communicated to the team or should it be correct forestClientNumber before user can go next? 
Basically, this becomes a typical "Form Validation" question.